src/components/Organisms/index.jsx

```diff
@@ -0,0 +1,75 @@
+import React, { useRef, useState, useEffect } from "react";
+import styled from "styled-components";
+import FONTFAMILY from "../../variables/font_family";
+import TEXTS from "../../variables/texts";
+import COLOR from "../../variables/color";
+import AddTaskButton from "../Atoms/AddTaskButton/index";
+import Tasks from "../Molecules/index";
+
+const TodoCard = () => {
+  const [taskList, setTaskList] = useState([{ name: "", initializing: true }]);
+
+  const onAddTaskButtonClick = () => {
+    setTaskList([...taskList,({ name: "", initializing: true })]);
+  };
+
+  const onTaskComplete = ({ index }) => {
+    const deleteList = [...taskList];
+    //console.log(index);
+    deleteList.splice(index, 1);
+    setTaskList(deleteList);
+    //setTaskList([[...taskList].slice(0, index), [...taskList].slice(index+1)])
+  };
+
+  const onTaskNameChange = ({ value, index }) => {
+    const changeList = [...taskList];
+    //console.log(value+" "+index);
+    if (value) {
+      //changeList[index].name = value;
+      changeList.splice(index, 1,{ name: value, initializing: true })
+    } else {
+      changeList.splice(index, 1);
+    }
+    setTaskList(changeList);
+  };
+
+  return (
+    <StyledWrapper>
+      <AddTaskButton onClick={onAddTaskButtonClick} />
+      <StyledTaskList>
+        {taskList.map((task, index) => {
+          return (
+            <Tasks
+              key={index}
+              onTaskChange={(value) => {
+                onTaskNameChange({ value, index });
+              }}
+              onTaskComplete={(index) => {
+                onTaskComplete(index);
```

Comment:
```suggestion
              onTaskComplete={() => {
                onTaskComplete({index});
```
こうでした🙏

`onTaskComplete` に渡す関数は、 `Tasks` コンポーネント内部で  `button` 要素の `onClick` 属性に渡されているため、第一引数にはイベントオブジェクトが入ってきてしまっていました。
41行目のindexを使うようにしましょう。

![image](https://github.com/JunPlayGround/Jun-frontend-basic-2023-todoapp/assets/83479175/27f7d43f-593f-49a0-9ec1-df67eca176ec)
（ログ出力結果）